marklogic-data-hub-central/ui/src/components/load-data/new-data-load-dialog/new-data-load-dialog.test.tsx

```diff
@@ -33,7 +33,16 @@ describe('New/edit load data configuration', () => {
     expect(baseElement.querySelector('#otherSeparator')).not.toBeInTheDocument();
     expect(baseElement.querySelector('#fileUpload')).toBeInTheDocument();
     expect(baseElement.querySelector('#targetFormat')).toBeInTheDocument();
-    expect(baseElement.querySelector('#outputUriReplacement')).toBeInTheDocument();
+    expect(baseElement.querySelector('#outputUriReplacement')).not.toBeInTheDocument();
```

Comment:
Generally, I wouldn't bother with these assertions that verify that things were removed. Someone will wonder 6 months later - why is it important that we assert this? I think it's sufficient to just remove the feature.